modules/core/src/main/java/org/apache/fluo/core/log/TracingTransaction.java

```diff
@@ -96,6 +96,11 @@ private String encC(Map<Column, Bytes> ret) {
         + "=" + enc(e.getValue())));
   }
 
+  private String encMBC(Map<Bytes, Set<Column>> m) {
```

Comment:
New private method with obfuscated name should have some comment describing what it's supposed to do.